spyder/app/mainwindow.py

```diff
@@ -1112,16 +1113,17 @@ def __getattr__(self, attr):
             pass
         return super().__getattr__(attr)
 
-    def update_lsp_logs(self):
-        """Create an action for each lsp log file."""
-        self.menu_lsp_logs.clear()
-        lsp_logs = []
+    def update_debug_logs(self):

```

Comment:
I have to put this in the plugin rather than the container in order to access the Editor plugin for menu item triggered actions. We should talk about this.